Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3922,8 +3925,13 @@ def panorama_register_user_tag_command(args: dict):
         raise Exception('The panorama-register-user-tag command is only available for PAN-OS 9.X and above versions.')
     tag = args['tag']
     users = argToList(args['Users'])
+    # if not given, timeout will be 0 (never expires)
+    timeout = int(args.get('timeout', '0'))
+
+    if major_version <= 8 and timeout:
+        raise DemistoException('The timeout argument is only applicable on 9.x PAN-OS versions or higher.')
```

Comment:
This entire command is only supported on 9.x, see the code in line 3924, hence this check is redundent

Reply:
Agreed, removing.